azure-quantum/azure/quantum/target/target.py

```diff
@@ -327,6 +327,52 @@ def _get_azure_target_id(self) -> str:
     def _get_azure_provider_id(self) -> str:
         return self.provider_id
 
+    def _qir_module_to_gates(self, qir_module) -> List[int]:
```

Comment:
Regarding this method, there are a couple of things that could be changed:
- It should probably be a class method.
- Consider returning a data class object. Returning a list makes it difficult to figure out what is the meaning of a number at a particular index is, and it also makes its use more error prone.
- Consider changing to name to something that reflects more accurately what is calculates like `_calculate_qir_module_gate_stats`.

Reply:
good suggestion!